.github/workflows/ci-core.yml

```diff
@@ -136,11 +154,15 @@ jobs:
           - cmd: go_core_ccip_deployment_tests
             os: ubuntu22.04-32cores-128GB
             printResults: true
+          - cmd: go_core_scripts_tests
+            if: ${{ needs.filter.outputs.should-run-ci-core == 'true' }}
+            os: ubuntu22.04-8cores-32GB
+            printResults: true
```

Comment:
I think this should be it's own job. Putting this into this matrix will subject it to 1.5-2 minutes of unneeded setup everytime this is run.

ci-scripts doesn't depend on the same environment information as core/deployment/fuzz/race testing.